main/src/library/Fixpoint3/TypeInfo.flix

```diff
@@ -118,7 +118,8 @@ mod Fixpoint3.TypeInfo {
             };
             foreach (id <- Vector.range(0, maxId)) {
                 let curArr = Array.get(id, arr);
-                foreach (index <- Vector.range(0, Array.length(curArr))) match MutDisjointSets.find((Int32.toInt64(id), index), typeUnifier) {
+                let range = Vector.range(0, Array.length(curArr));
+                foreach (index <- range) match MutDisjointSets.find((Int32.toInt64(id), index), typeUnifier) {
```

Comment:
I think its body expression that trips me up. Could you curly brace and new line its body?